src/python/zquantum/core/estimator.py

```diff
@@ -215,18 +215,12 @@ def get_estimated_expectation_values(
             frame_circuits, measurements_per_frame
         )
 
-        expectation_values_set = []
-        for frame_operator, measurements in zip(frame_operators, measurements_set):
-            expectation_values_set.append(
-                expectation_values_to_real(
-                    measurements.get_expectation_values(frame_operator)
-                )
-            )
-
-        if target_operator.terms.get(()) is not None:
-            expectation_values_set.append(
-                ExpectationValues(np.array([target_operator.terms.get(())]))
+        expectation_values_set = [
```

Comment:
This is done properly in new Estimators, but I didn't want to overcomplicate this logic since this code will be depreciated in two weeks or so anyway.
And since I don't think it's a big deal, I left it as it is.